node/node.go

```diff
@@ -50,6 +50,10 @@ var (
 	// V4OPRUpdate indicates the activation of additional currencies and ecdsa keys.
 	// Estimated to be  Feb 12, 2020, 18:00 UTC
 	V4OPRUpdate uint32 = 231620
+
+	// V5OPRUpdate indicates the activation of additional currencies and ecdsa keys.
+	// Estimated to be  May 11, 2020, 18:00 UTC
+	V5OPRUpdate uint32 = 244970
```

Comment:
Same as in the other PR (https://github.com/pegnet/pegnet/pull/359#discussion_r422632825), this will need to be changed.

Reply:
@WhoSoup , I have replaced the V5HeightActivation as TBD placeholder value (999999).
It would be better you decide the correct value for me.